File: 02 Digits and Numbers - Foundation Programs/program-13.cpp
// Write a program that takes a 3-digit number as input and checks if it is a palindrome. Example: 121 →
// Yes, 123 → No. Display "Palindrome" or "Not Palindrome"


#include<iostream>
using namespace std;
int main()
{
    int number,rev_digit;
    cout<<"Enter 3-digit number : ";
    cin>>number;
    
    rev_digit=(number%10)*100;  //123 as input number we got 3*100 = 300
    rev_digit=rev_digit+((number/10)%10)*10;
    rev_digit=rev_digit+(number/100);
    if (rev_digit==number)
    {
        cout<<"Palindrome.";
    }else{
        cout<<"Not Palindrome.";
    }
    
    return 0;
}

File: archit_vishwakarma/Level-1/03-Digits-And-Numbers/program-13.py
n = input("\nEnter a number : ")

try:
    n = int(n)
    if n < 1000 and n > 99:

        a = n//100
        b = n%10
        result = a-b

        z = (abs(result)+8)//9
        print(f"digit is : {'Not '*z + 'Palindrome' }\n")

    else:
        print("Enter a THREE Digit Number")
except ValueError:
    print("enter a valid number")